main/views.py

```diff
@@ -1,118 +1,13 @@
 from django.shortcuts import render, redirect, reverse
-from django.core.mail import send_mail
-from main.models import UserProfile, Stock, Transaction, NewsPost
+from main.models import UserProfile, Stock, Transaction, StockPurchased, NewsPost
 from django.http import HttpResponse, HttpResponseRedirect
-from django.contrib.auth import authenticate, login, logout
 from django.contrib.auth.models import User
 from django.contrib.auth.decorators import login_required
 from django.conf import settings
 import json
 import re
 from django.db.models import F
 
-special_character_regex = re.compile(r'[@_!#$%^&*()<>?/\|}{~:]')
```

Comment:
Line 13 to 115 removed as not authentication functions are not required . AllAuth handles them